docs/readme-file-template.md

```diff
@@ -4,7 +4,9 @@ title: Readme file template
 sidebar_label: Readme File Template
 ---
 
-README is a file that brings a little description about the whole project, which contains an introduction and explanation of this one. The kind of information that can be describe here is information requeried to understand the project and its structure or purpose. Then, we will give a template to create a README file for a new project. To copy this readme markdown source code, [please follow this link](readme-file-source-code-template).
+README is a file that brings a little description about the whole project, which contains an introduction and explanation of this one. The kind of information that can be describe here is information requeried to understand the project and its structure or purpose. Then, we will give a template to create a README file for a new project.
+
+### To copy this readme markdown source code, [please follow this link](readme-file-source-code-template)
```

Comment:
### To copy this readme template as markdown source code click here

Reply:
Working on